<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QWidget" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BlackJack VM</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hitButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>70</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: orange;
color:white;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>Hit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="standButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: orange;
color:white;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>Stand</string>
   </property>
  </widget>
  <widget class="QLabel" name="Score">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Bet">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dScore">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="coin5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>210</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/images/PlayingCards/5.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="coin20">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>210</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/images/PlayingCards/20.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="coin50">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>210</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/images/PlayingCards/50.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="coin100">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>210</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/images/PlayingCards/100.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="coin500">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/images/PlayingCards/500.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="coin1000">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/images/PlayingCards/1K.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dealButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: orange;
color:white;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>DEAL</string>
   </property>
  </widget>
  <widget class="QLabel" name="link">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>410</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="betLabel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="betValue">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <action name="actionquit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
